src/functions/node.php

```diff
@@ -21,3 +24,13 @@ function node_view($node, $view_mode = 'full', $langcode = null)
       ->getViewBuilder('node')
       ->view($node, $view_mode, $langcode);
 }
+
+function node_types_rebuild(): void
+{
+    $tags = \Drupal::config('node_type')->getCacheTags();
+    $tags[] = 'config:node_type_list';
+    Cache::invalidateTags($tags);
```

Comment:
I'm not sure if this is correct. We probably want the value of `getListCacheTags` from the `node_type` entity type definition.